docs/examples/201/policy-definition-with-assignment/main.bicep

```diff
@@ -0,0 +1,77 @@
+targetScope = 'subscription'
+
+param listofAllowedLocations array {

```

Comment:
This is more a style/taste nit, so not a required change. Since this `param` only has a `default` modifier, you can use this syntax:

```
param listofAllowedLocations array = [
  'norwayeast'
  'westeurope'
]
```